_posts/2019-03-28-smart-proxy-capabilities-explained.md

```diff
@@ -0,0 +1,205 @@
+---
+layout: post
+title: "Foreman Proxy Registration Protocol v2 explained"
+author: "Ewoud Kohl van Wijngaarden"
+date: "28 March 2019"
+tags:
+  - foreman
+  - foreman-proxy
+  - smart-proxy
+---
+
+# Foreman Proxy Registration Protocol v2 explained
+
+In Foreman 1.22 the Proxy registration protocol changed. This blog takes a look at how it was, how it'll be and why this is an improvement.
+
+## Classic Foreman Proxy registration
+
+Before diving in, there are some things one must know. First of all, there is our main application Foreman. For this it doesn't matter which plugins are installed, including Katello. It is a [Ruby on Rails] application. It's typically deployed on `https://foreman.example.com`. Some features also need `http://foreman.example.com` but that's out of scope for this blog. When using the foreman-installer it defaults to using the Puppet CA for certificates, but this is out of historical convenience. In the Katello scenario a custom CA implementation is used instead.
```

Comment:
It wasn't clear that by example you are actually talking about https vs http. Perhaps explicit mention of the protocol would be better. Something like It's typically consumed via HTTPS. Some features also need access over HTTP but ...